### DISCUSSION TITLE
Is it possible to set transparent?

### DISCUSSION THREAD

**Discussion by byte2pixel** [rmehri01/onenord.nvim#76] [Q&A] [answered]
I am wondering if it is possible to have transparency?

I tried 
```lua
disable = {
  background = true,
}
```
But that didn't seem to do anything.

I have used gruvbox and a few other theme's and they did support transparency.   I liked this color scheme a bit more and was hoping I could get a similar setup.

Example:
![image](https://github.com/rmehri01/onenord.nvim/assets/31562317/6c522b98-06be-419b-a2ee-3c60cefbb97d)

**Comment by rmehri01** [comment, marked as answer]
Hey! That's odd, the disable background option seems to work for me:

<img width="1470" alt="Screenshot 2023-10-22 at 11 18 12 AM" src="https://github.com/rmehri01/onenord.nvim/assets/52933714/703ddb67-91d6-4036-bca6-8f91a2fabccc">

Seems unlikely but maybe it's a configuration issue? I could try reproducing it on my machine

Edit: nevermind, I tried using your config and it seems to work fine for me, do you have an idea of anything else that could be causing it? Could you also link the themes that work for you so I could take a look at them?

**Reply by byte2pixel** [reply]
I'm a dirty liar.  I have it working now.  Helps if I spell disable right.

My colorscheme.lua  I guess `diable = true` isn't right.... Sorry.  I swear it was right.  I'm fairly new to using nvim so maybe I accidentally deleted it when saving.  I'm on github on a different machine.

```lua
return {
  {
    "rmehri01/onenord.nvim",
    lazy = false,
    opts = {
      theme = "dark",
      fade_nc = true,
      disable = {
        background = true,
      },
    },
  },
  {
    "LazyVim/LazyVim",
    opts = {
      colorscheme = "onenord",
    },
  },
}
```

working fine now, just need to see how to make floating windows and stuff transparent.  This is working for the editor now.  Thanks.

**Reply by rmehri01** [reply]
Hahaha no problem! Actually I think I should make the floating windows transparent by default too in this mode (such as for telescope) so I'll make some changes and let you know when they're up :)

**Reply by byte2pixel** [reply]
Yes I noticed in Telescope and my floating lazygit terminal window it was not transparent.

When it was broken (from the typo) i had been able to get the normal window transparent by adding 
```lua
vim.api.nvim_set_hl(0, "Normal", { bg = "none" })
```
I removed that line after finding the typo and it still achieved the same effect.  Look forward to testing it out.  I would help if I knew the slightest about neovim plugins and this sort of thing but I don't.

**Reply by rmehri01** [reply]
Don't worry testing is already really helpful :)

I added a new config option in https://github.com/rmehri01/onenord.nvim/commit/c1d18ba899af25e4179cad6d401fe933612638bf, so you can now do:

```lua 
disable = {
  background = true,
  float_background = true,
}
```

which should make all the floats transparent. Let me know if you run into any issues!

**Reply by byte2pixel** [reply]
It definitely made the floats transparent.  The little LazyVim notifications are now transparent.
Is it possible to do the same for terminals?  I pushed my current config: https://github.com/rupture-ttv/neovim-config  it is LazyVim with just some small changes.  I know LazyGit opens in a terminal so I am guessing that background is a different group or something?

![image](https://github.com/rmehri01/onenord.nvim/assets/31562317/1198f1eb-cbf5-4042-bb9c-88b97b112906)

**Reply by byte2pixel** [reply]
I don't typically have the Neo-tree open either I'm guessing that is something special too.  Not worried about either LazyGit or the tree window.  Just curious if it is possible.  You've done enough help.  Really enjoy the colorscheme.

**Reply by byte2pixel** [reply]
just realized it didn't pull in the update yet... my mistake.

**Reply by byte2pixel** [reply]
Ok yeah it works now!

![image](https://github.com/rmehri01/onenord.nvim/assets/31562317/ca16a811-e4b9-47e1-b004-9228c1c10996)

**Reply by rmehri01** [reply]
Oh yeah whoops, I'll add neotree too. Glad to see it's working :)

**Reply by byte2pixel** [reply]
just checked out the latest seems to be working great!

![image](https://github.com/rmehri01/onenord.nvim/assets/31562317/6f580c82-aee4-4270-a436-8cd5a9dbb497)

**Reply by rmehri01** [reply]
Nice! :smile:
